fooshared/tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "./src",
     "module": "commonjs",
     "declaration": true,
     "target": "es2020",
-    "outDir": "./dist"
+    "composite": true
```

Comment:
This indicates to the TypeScript compiler that this packages will be part of a project composed of multiple packages.